avro-fastserde/src/main/java/com/linkedin/avro/fastserde/PrimitiveFloatList.java

```diff
@@ -59,6 +56,7 @@ public PrimitiveFloatList(Collection<Float> c) {
       elements = new float[c.size()];
       addAll(c);
     }
+    byteBuffer = new CompositeByteBuffer();
```

Comment:
Since we know we won't use it, can we initialize it with an empty list in `CompositeByteBuffer` for this scenario?
`Collections.emptyList`.

Reply:
> Since we know we won't use it, can we initialize it with an empty list in `CompositeByteBuffer` for this scenario?
> `Collections.emptyList`.

Sure.